ts/torch_handler/object_detector.py

```diff
@@ -17,11 +18,12 @@ def __init__(self):
 
     def initialize(self, ctx):
         super(VisionHandler, self).initialize(ctx)
-        self.initialized = False
-        self.device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
-        self.model.to(self.device)
-        self.model.eval()
-        self.initialized = True
+        if torchvision_version == "0.5.0":
```

Comment:
My understanding is that this bug will also strike users on TorchVision 0.4. Could we change this condition to demand a version < 0.6?

Thoughts:
* We don't want to do a string comparison, because that would fail if we get to version 0.10.0.
* I've done some digging, and PyTorch & TorchVision version strings are consistently three integers separated by periods. So if we were to split the version string on the periods, and treat the components like ints, that would look something like:

```
if major_version <= 0 and minor_version < 6:
    ...
```

Reply:
(NB: I don't encourage treating versions strings numerically in the general case, but this is a narrow, bounded case with low risk that serves our need here.)

Reply:
Done.